packages/devtools_app/lib/src/memory/memory_vm_chart.dart

```diff
@@ -150,12 +164,11 @@ class MemoryVMChartState extends State<MemoryVMChart> with AutoDisposeMixin {
 
   // TODO(terry): Move colors to theme?
   final capacityColor = Colors.grey[400];
-  //static const usedColor = Color.fromARGB(255, 51, 181, 229);
-  static const usedColor = Color(0xff33b5e5); // HoloBlue
+  static const externalColor = Color(0xff42a5f5); // Color.blue[400]
```

Comment:
Nit: if this is Color.blue[400] why not just make this be
static final externalColor = Color.blue[400]
?

Reply:
static final capacityColor = Colors.grey[400];
  static final externalColor = Colors.blue[400];

Done.